### PR TITLE
Fixed Issue (#1631). added hidden class to the div which earlier just had custom-menu-content class

### PR DESCRIPTION
**Pull request by tushargr**
… had custom-menu-content class

Fixes #[1631. Note: This will automatically close the issue. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]

Changes: [Added "hidden" class to the div which earlier just had one class- "custom-menu-content" in index.html    deployment link- https://warm-headland-17924.herokuapp.com/]

Screenshots for the change:

### CODE REVIEW

**Comment by tushargr** [comment]
My deployment link- https://warm-headland-17924.herokuapp.com/
The menu bar button is now working correctly.

File: src/www/index.html [hunk 1]
```diff
@@ -168,7 +168,7 @@
   <div class="custom-menu-cont hidden">
     <div class="custom-menu">
       <div class="hide-arrow"></div>
-      <div class="custom-menu-cont">
+      <div class="custom-menu-cont hidden">
```
Comment (geekyd) [inline src/www/index.html]: Remove `custom-menu-cont` from here.

**Comment by tushargr** [comment]
@geekyd if i remove the hidden the toggle in form.js won't work. Actually initially both are hidden, when someone clicks the button, both div get toggled and thus become visible. Isn't it the required behavior

File: src/www/index.html (hunk 1 above)
Comment (geekyd) [reply src/www/index.html]: @tushargr Try removing both `hidden` & `custom-menu-cont` class

**Pull request closed without merging**